bin/spice/pkg/github/github.go

```diff
@@ -47,8 +48,9 @@ func NewGitHubClientFromPath(path string) (*GitHubClient, error) {
 
 func NewGitHubClient(owner string, repo string) *GitHubClient {
 	return &GitHubClient{
-		Owner: owner,
-		Repo:  repo,
+		Owner:       owner,
+		Repo:        repo,
+		accessToken: os.Getenv("SPICE_CLI_GITHUB_TOKEN"),
```

Comment:
Let's just call this `SPICE_GITHUB_TOKEN`. We will also need to support disabling this.

Reply:
reworked to use GH_TOKEN or GITHUB_TOKEN as most commonly used